### PR TITLE
KEYCLOAK-11372 Support for attestation statement verification

### PR DESCRIPTION
**Pull request by tnorimat**
This documentation PR for https://github.com/keycloak/keycloak/pull/6449.

### CODE REVIEW

**Review by mposolda — DISMISSED** [review]
@tnorimat Thanks, Looks good to me. I am adding @andymunro as a reviewer too.

File: server_admin/topics/authentication/webauthn.adoc [hunk 1]
```diff
@@ -161,7 +160,7 @@ The configurable items and their description follow.
  For more details, see https://www.w3.org/TR/webauthn/#rp-id[WebAuthn Specification].
 
 |Attestation Conveyance Preference
-|It tells the WebAuthn API implementation on the browser (https://www.w3.org/TR/webauthn/#webauthn-client[WebAuthn Client]) the preference of how to generate an Attestation Statement. However, {project_name} does not verify the Attestation Statement so that only "none" option can be selected. This is an optional configuration item that is applied to the operation of registering WebAuthn authenticator. If no option is selected, its behavior is the same as selecting "none".
+|It tells the WebAuthn API implementation on the browser (https://www.w3.org/TR/webauthn/#webauthn-client[WebAuthn Client]) the preference of how to generate an Attestation Statement. This is an optional configuration item that is applied to the operation of registering WebAuthn authenticator. If no option is selected, its behavior is the same as selecting "none".
```
Comment (andymunro) [inline server_admin/topics/authentication/webauthn.adoc]: Should be:  registering "a" WebAuthn authenticator.

File: server_admin/topics/authentication/webauthn.adoc [hunk 2]
```diff
@@ -187,3 +186,9 @@ The configurable items and their description follow.
 |The white list of AAGUID of which a WebAuthn authenticator can be registered. This is applied to the operation of registering WebAuthn authenticator. If no entry is set on this list, any WebAuthn authenticator can be registered.
 
 |===
+
+==== Attestation Statement Verification
+
+When registering a webauthn authenticator, {project_name} verifies an attestation statement generated by this webauthn authenticator. On this verification process, {project_name} validates this attestation statement's trustworthiness. It requires trust anchor's certificates. {project_name} uses the link:{installguide_truststore_link}[{installguide_truststore_name}]  so that you need to import these certificates onto it in advance.
```
Comment (andymunro) [inline server_admin/topics/authentication/webauthn.adoc]: Please use a consistent capitalization.   I believe you are using  WebAuthn  not webauthn

**Review by andymunro — DISMISSED** [review]
Minor changes.  Otherwise, looks good.

File: server_admin/topics/authentication/webauthn.adoc (hunk 1 above)
Comment (tnorimat) [reply server_admin/topics/authentication/webauthn.adoc]: Yes, I'll fix it as you said.

File: server_admin/topics/authentication/webauthn.adoc (hunk 2 above)
Comment (tnorimat) [reply server_admin/topics/authentication/webauthn.adoc]: I see, I'll use WebAuthn.

**Comment by tnorimat** [comment]
@andymunro Thank you for your review.
@mposolda Please note that this PR affects https://github.com/keycloak/keycloak-documentation/pull/780 which also modifies webauthn.adoc.

**Comment by mposolda** [comment]
@tnorimat Do you have a chance to fix the conflict? Thanks

**Comment by tnorimat** [comment]
@mposolda I've resolved the conflict.

**Review by mposolda — APPROVED** [review]
@tnorimat Thanks!

**Pull request merged**